TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/BasicMotorTest.java

```diff
@@ -10,35 +10,44 @@
 /**
  * TeleOp OpMode for testing motors.
  */
+// "BasicMotorTest" is the name of OpMode in Driver Station
 @TeleOp(name = "BasicMotorTest", group = "Iterative OpMode")
+
```

Comment:
Remove the newline here. It won't fuck anything up, but it's makes things a bit unclear nonetheless.